### DISCUSSION TITLE
Frontend initialization of electron app fails on Windows when runing the exe via UNC path.

### DISCUSSION THREAD

**Discussion by aghayanlena** [eclipse-theia/theia#13510] [General] [answered]
The initialization of my theia custom electron application fails on Windows when running theia via UNC path. ( --no-sandbox option is presented)
From the Network tab of DevTools the following request is refused: 
Request URL: http://**<server_name_of_UNC_paht>**/socket.io/?EIO=4&transport=polling&t=OvSpqm9

Although, from the logs, it is listening to http://**127.0.0.1**:<port_number> . and the response of the same request to localhost  is 
$ curl http://127.0.0.1:<port_number>/socket.io/?EIO=4\&transport=polling
0{"sid":"cx7PqAU7lTT64JufAAAB","upgrades":["websocket"],"pingInterval":30000,"pingTimeout":60000,"maxPayload":100000000}

is this behavior expected? is there any configuration that I should apply when running the electron app via the UNC path? 

Note: I don't see any issue on Windows when running theia with a local path.

**Comment by msujew** [comment, marked as answer]
Hey @aghayanlena,

This is a known issue: https://github.com/eclipse-theia/theia/issues/11997. Theia currently does not support opening workspaces from network drives.

**Reply by aghayanlena** [reply]
Hi @msujew , 

My issue is running the theia executable from the UNC path(the workspace file is not provided). In this case, the front-end does not open successfully; it keeps sending the socket.io request forever.  

Is there a hotfix that we can apply?

**Reply by msujew** [reply]
Oh, I see. I misread your original post. I assume that's a bug in our messaging layer that messes up when trying to build the server URL. Please create a new bug report for this.

**Reply by aghayanlena** [reply]
Created new Bug report:  #13512

**Reply by aghayanlena** [reply]
Hi @msujew ,

My project has a dependency on the #13512 issue, and currently, I don't have a workaround for it. Are there any plans for this issue? 

Thanks,
Lena

**Reply by msujew** [reply]
@aghayanlena I've just labeled the issue to make it clear that anyone can pick it up. I don't think that any active contributor party has this on their roadmap. Feel free to contribute something (and you can likely expect a timely review), but don't expect someone else to fix this on their own time/cost.

**Reply by aghayanlena** [reply]
Ok. We will consider fixing this from our side.

Thanks @msujew !
